pkg/controllers/dynakube/logmonitoring/daemonset/volumes.go

```diff
@@ -74,7 +77,7 @@ func getDTVolumes() []corev1.Volume {
 			VolumeSource: corev1.VolumeSource{
 				HostPath: &corev1.HostPathVolumeSource{
 					Path: dtLibVolumePath,
-					Type: ptr.To(corev1.HostPathDirectoryOrCreate),
+					Type: ptr.To(corev1.HostPathDirectory),
```

Comment:
is this OK to do?

as this is for the dynatrace logs, so it can be not there without us

Reply:
I agree I changed it back.

@chrismuellner do you also agree?

done https://github.com/Dynatrace/dynatrace-operator/pull/4413/commits/226f5a9a17bf7b71a17fec9dd75a90d322124032

Reply:
yes, directories for log agent specific configuration should be created if they don't exist!